corehq/apps/app_manager/tests/views/test_release_build.py

```diff
@@ -0,0 +1,65 @@
+from django.test import TestCase
+from django.test.client import Client
+from django.urls import reverse
+
+from corehq.apps.app_manager.tests.app_factory import AppFactory
+from corehq.apps.app_manager.tests.util import get_simple_form, patch_validate_xform
+
+from corehq.apps.app_manager.models import ApplicationReleaseLog
+from corehq.apps.domain.models import Domain
+from corehq.apps.users.models import WebUser
+from corehq.util.test_utils import flag_enabled
+
+
+class TestReleaseBuild(TestCase):
+    @classmethod
+    @patch_validate_xform()
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.client = Client()
+
+        cls.domain_name = "fandago"
+        cls.domain = Domain.get_or_create_with_name(cls.domain_name, is_active=True)
+
+        cls.username = 'bananafana'
+        cls.password = '*******'
+        cls.user = WebUser.create(cls.domain.name, cls.username, cls.password, None, None, is_admin=True)
+        cls.user.eula.signed = True
+        cls.user.save()
+
+        factory = AppFactory(cls.domain_name, name="cheeto")
+        m0, f0 = factory.new_basic_module("register", "cheeto")
+        f0.source = get_simple_form(xmlns=f0.unique_id)
+        cls.app = factory.app
+        cls.app.target_commcare_flavor = 'commcare_lts'
+        cls.app.save()
+
+        cls.app_build = cls.app.make_build()
+        cls.app_build.save()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.user.delete(cls.domain.name, deleted_by=None)
+        cls.domain.delete()
```

Comment:
does this include app and app build delete as well? I am not sure if we delete apps separately in tests as well.

Reply:
https://github.com/dimagi/commcare-hq/blob/d416577393329e6ad00e0e4e94fd1376c28355c6/corehq/apps/domain/models.py#L796

confirmed that these are handled as a part of delete function call on Domain object